src/ss_ros.py

```diff
@@ -50,15 +50,18 @@ def __init__(self, ros_node):
 
         # set up logger
         self.logger = logging.getLogger(__name__)
-        self.logger.info("Subscribing to topics: opal_tablet_action, " +
```

Comment:
This log message was in a good location, since it was logging that the node will be trying to subscribe to topics next... why'd you move it to after the subscription happens? Also if you're changing the topic names, please update this log message to list the new topic names.